velox/common/serialization/Serializable.h

```diff
@@ -177,6 +177,13 @@ class ISerializable {
           registryWithContext.Create(name, obj, context));
     }
 
+    const auto& registryOfUniquePtr =
```

Comment:
Can we move this to a separate function like `std::unique_ptr<T> deserializeAsUniquePtr(const folly::dynamic& obj, void* context)`?

Reply:
OK, done.